jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/Resource.java

```diff
@@ -55,148 +46,22 @@
  * a file, a URL or an entry in a jar file.
  * </p>
```

Comment:
Alternative javadoc ...

```
<p>
A Resource is a {@link Path} object pointing to a File or Directory that can be represented by a Java {@link FileSystem}.
</p>
<p>
Supports real FileSystems, and also <a href="https://docs.oracle.com/en/java/javase/17/docs/api/jdk.zipfs/module-summary.html">ZipFS</a>.
</p>
```

Reply:
I've used your javadoc but slightly adapted it.